db/migrate/20220715230147_create_users.rb

```diff
@@ -0,0 +1,18 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :first_name, null: false 
+      t.string :last_name, null: false
+      t.string :email, null: false, index: { unique: true, name: 'unique_emails' }
```

Comment:
es atributo `name` no sería necesario en este caso.
```suggestion
      t.string :email, null: false, index: { unique: true }
```